core/src/main/java/org/apache/jsieve/tests/Header.java

```diff
@@ -199,10 +202,10 @@ protected boolean match(String comparator, String matchType,
             }
         // Iterate over the header values looking for a match
         boolean isMatched = false;
-        Iterator headerValuesIter = headerValues.iterator();
+        Iterator<String> headerValuesIter = headerValues.iterator();
         while (!isMatched && headerValuesIter.hasNext()) {
```

Comment:
I know peple don't like it but I think a for with a return in it when you find the result is way more consise